front-end/src/components/UploadForm.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+
+function UploadForm(){
+  const [verificationToken, setVerificationToken] = useState("");
+  function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+    console.log(verificationToken);
+    if((verificationToken.match(/^[A-Z]+$/) && verificationToken.length === 4)){
+      //send token to backend
+    }
+    else {
+      //display error pop-up message
+    }
+  }
+  
+  return (
+    <div>
+      {/**Header section*/}
+      <section className="section is-medium has-text-centered">
+        <div className="container">
+          <h1 className="title">Sharing my close contacts with healthcare professional</h1>
```

Comment:
[master L12](https://github.com/JoshuaBuditama/slowvid/blob/master/front-end/src/components/Notifications.tsx#L12) uses the term 'health care' (note space)